includes/services/wsl.authentication.php

```diff
@@ -438,69 +438,6 @@ function wsl_process_login_get_user_data( $provider, $redirect_to )
 		}
 	}
 
-	// because instagram doesn't (do any?) have an email, we need to check if the option "require email" is set and then get the email from
```

Comment:
@miled please check the re-order I made in this function logic. Actually, I followed what you describe in function comment. For some reason, step 2 has moved to 5 and 4 had been removed. This was breaking some logic with account linking. I tested it and it works fine now (also the bouncer rules), and it makes sense like this. Let me know if I have to check anything more.